app/models/dba/partition_all.rb

```diff
@@ -4,31 +4,25 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
+# frozen_string_literal: true
+
 class Dba::PartitionAll
   def tables
-    t = HmisCsvImporter::Loader::Loader.loadable_files.values.map(&:table_name)
-    t += HmisCsvImporter::Importer::Importer.importable_files.values.map(&:table_name)
-    t.reject! do |x|
+    t = {}
+    loader_table_classes.each do |klass|
+      t[klass] = klass.table_name
+    end
+    importer_table_classes.each do |klass|
+      t[klass] = klass.table_name
+    end
+    t.reject! do |_klass, x|
       x.match?(/exports|organizations|projects|funders|inventories|project_cocs|affiliations|users/)
     end
     t.sort
```

Comment:
maybe could be golfed down (untested). Looks like this is returning an array (hash.sort)
```suggestion
    (
      loader_table_classes + 
      importer_table_classes
    ).uniq.sort.filter_map do |klass|
      next if klass.table_name.match?(/exports|organizations|projects|funders|inventories|project_cocs|affiliations|users/)
      [klass, klass.table_name]
    end
```